Subject: various POD fixes
Bug: https://rt.cpan.org/Ticket/Display.html?id=88499
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=88499
Author: Jonathan Yu <jawnsey@cpan.org>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>,
 gregor herrmann <gregoa@debian.org>
Last-Update: 2022-04-21

--- a/bin/prove
+++ b/bin/prove
@@ -78,7 +78,7 @@ Options that take arguments:
 
 =head1 NOTES
 
-=head2 .proverc
+=head2 F<~/.proverc> / F<./.proverc>
 
 If F<~/.proverc> or F<./.proverc> exist they will be read and any
 options they contain processed before the command line options. Options
