Description: POD spelling
 Multple → Multiple and arbitarily → arbitrarily
Bug: https://rt.cpan.org/Ticket/Display.html?id=124816
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=124816
Author: Damyan Ivanov <dmn@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2022-04-21

--- a/lib/TAP/Parser/Scheduler.pm
+++ b/lib/TAP/Parser/Scheduler.pm
@@ -58,7 +58,7 @@ The "C<rules>" data structure is the the
 to express simple rules like "run all tests in sequence" or "run all tests in
 parallel except these five tests.". However, the rules structure also supports
 glob-style pattern matching and recursive definitions, so you can also express
-arbitarily complicated patterns.
+arbitrarily complicated patterns.
 
 The rule must only have one top level key: either 'par' for "parallel" or 'seq'
 for "sequence".
